hqscripts/management/commands/copy_data_to_couch3.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand
+from cloudant.client import Cloudant
+
+COUCH2_HOST = "localhost"
+COUCH2_PORT = 5984
+
+COUCH3_HOST = "localhost"
+COUCH3_PORT = 5985
+
+
+def sync_dbs(source_db_obj, destination_db_obj):
+    for doc in source_db_obj:
+        doc_obj = destination_db_obj.create_document(doc)
+        if not doc_obj.exists():
+            print(f'Unable to sync ${doc_obj._id} in ${destination_db_obj.database_name}')
+
+
+class Command(BaseCommand):
+    help = """Script to copy local couchdb2 docs to couchdb3.
+    It will iterate through all dbs sequentially and populate in couchdb3"""
```

Comment:
Would just copying the underlying data directory to the new location work too? At least for me (and I think it's the default setup) all the docker files are stored in `ls ~/.local/share/dockerhq/`

Reply:
Yep. We can do that it will work fine for upgrading to couch 3 but I face an issue on my local system that when I tried to use the data of couch 3 in couch 2, that did not work well for me. So I wrote this script which worked both ways. 

We can give options either to copy the data to a new directory or to run this script.

Reply:
Gotcha okay that makes sense. I wasn't thinking about going back and forth between them, but that totally makes sense in a dev environment. Thanks for the explanation.